$NetBSD: patch-ae,v 1.1 2009/05/18 04:33:28 dholland Exp $

Don't treat version numbers as shell variables (formerly only in patch-ac).
Check for existence of mremap().

--- configure.ac.orig	2006-03-16 11:50:41.000000000 -0500
+++ configure.ac	2009-05-18 00:09:52.000000000 -0400
@@ -52,7 +52,7 @@ AC_CHECK_HEADERS([fcntl.h unistd.h stdli
 
 dnl Library functions.
 AC_FUNC_MALLOC
-AC_CHECK_FUNCS([floor gettimeofday memmove memset pow sqrt])
+AC_CHECK_FUNCS([floor gettimeofday memmove memset pow sqrt mremap])
 AC_CHECK_FUNC([munmap], lv_have_munmap=yes, lv_have_munmap=no)
 AC_FUNC_MMAP
 
@@ -127,7 +127,7 @@ if test "$ENABLE_PLUGIN_GDKPIXBUF" = "ye
   if test "$HAVE_GTK" = "yes"; then
     build_actor_plugins="$build_actor_plugins gdkpixbuf"
   else
-    AC_MSG_WARN([*** GTK+ >= ${gtk_required_version} is not found.
+    AC_MSG_WARN([*** GTK+ >= gtk_required_version is not found.
 	  	The libvisual GdkPixbuf image loader plugin won't be build.
 		GdkPixbuf is included within gtk-2.0 and newer, which can be
 		downloaded at http://www.gtk.org/])
@@ -148,7 +148,7 @@ if test "$ENABLE_GSTREAMER_PLUGIN" = "ye
   if test "$HAVE_GSTREAMER" = "yes"; then
     build_actor_plugins="$build_actor_plugins gstreamer"
   else
-    AC_MSG_WARN([*** GStreamer >= ${gst_required_version} is not found.
+    AC_MSG_WARN([*** GStreamer >= gst_required_version is not found.
 	  	The libvisual GStreamer viewer plugin won't be build.
 		GStreamer can be downloaded from http://www.gstreamer.org/])
   fi
